source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: post 
+title: "GSoC 2018: RubyPlot GR - A Scientific Plotting Library for Ruby Built on GR Framework"
+date: 2019-01-23 18:00
+comments: true 
+author: Pranav Garg 
+categories: [GSoC, GSoC 2018, RubyPlot, Data Visualization, Ruby, Matplotlib, GR Framework]
+---
+
+### 1. About
+
+For my summer of code Project I decided to create a plotting library.
+
+From the scratch.
+
+in Ruby.
+
+### 2. Application
+
+The GSoC 2018 application can be found [here](https://github.com/pgtgrly/Cairo_Graph/wiki/Google-Summer-of-Code-2018-Application).
```

Comment:
Please also add your proposal to this wiki: https://github.com/SciRuby/sciruby/wiki/Successful-GSOC-proposals

Reply:
Done